pepper-apis/src/test/java/org/broadinstitute/ddp/util/TestingUserUtil.java

```diff
@@ -176,9 +180,18 @@ private static CachedUser tryCachedUser(String userGuid, String auth0ClientId, S
             return null;
         }
 
-        DecodedJWT jwt = JWTConverter.verifyDDPToken(user.getToken(), JWTConverter.defaultProvider(auth0Domain));
-        if (jwt == null) {
-            LOG.warn("Unable to verify or decode jwt token for cached test user");
+        DecodedJWT jwt;
+        try {
+            jwt = JWTConverter.verifyDDPToken(user.getToken(), JWTConverter.defaultProvider(auth0Domain));
+            if (jwt == null) {
+                LOG.warn("Unable to verify or decode jwt token for cached test user");
+                return null;
+            }
+        } catch (TokenExpiredException e) {
```

Comment:
I find that when I come back the next day the cached token has expired, and it will stop the tests from running until I cleared the cached token. Figured out that we're not handle this exception so I added it.

Reply:
I run into this all the time while running tests locally. I had to go and delete that token manually. 
Thank you for fixing this.